src/main/resources/logback.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration>
+
+	<!-- Client logs -->
+	<include resource="org/springframework/boot/logging/logback/defaults.xml" />
+	<property name="LOG_FILE" value="${LOG_FILE:-${LOG_PATH:-${LOG_TEMP:-${java.io.tmpdir:-/tmp}}}/spring.log}" />
+	<include resource="org/springframework/boot/logging/logback/console-appender.xml" />
+	<include resource="org/springframework/boot/logging/logback/file-appender.xml" />
+	<root level="info">
+		<appender-ref ref="CONSOLE" />
+		<appender-ref ref="FILE" />
+	</root>
+
+	<!-- Advanced ICE logs -->
+	<appender name="FILE_ICE_ADVANCED"
+		class="ch.qos.logback.core.rolling.RollingFileAppender">
+		<encoder
+			class="ch.qos.logback.classic.encoder.PatternLayoutEncoder">
+			<Pattern>
+				%d{dd-MM-yyyy HH:mm:ss.SSS} %-5level - %msg%n
+			</Pattern>
+		</encoder>
+		<file>${ICE_ADVANCED_LOG}</file>
+		<rollingPolicy class="ch.qos.logback.core.rolling.TimeBasedRollingPolicy">
+			<cleanHistoryOnStart>true</cleanHistoryOnStart>
+			<fileNamePattern>${ICE_ADVANCED_LOG}.%d{yyyy-MM-dd}.gz}</fileNamePattern>
+			<maxHistory>7</maxHistory>
+			<totalSizeCap>30MB</totalSizeCap>
+		</rollingPolicy>
+	</appender>
+	<!-- This logger only logs to FILE_ICE_ADVANCED -->
+	<logger name="faf-ice-adapter-advanced" additivity="false" level="debug">
+		<appender-ref ref="FILE_ICE_ADVANCED" />
+	</logger>
+	<!-- This logger logs to both FILE_ICE_ADVANCED and normal log -->
```

Comment:
do we need this?

Reply:
We don't want to include the ice adapter log in the client log, it's pretty long.

Reply:
With this config, the STDOUT/ERR does not appear in the client log, but the logs from the `IceAdapterImpl` appear in the advanced log.
I thought it would be nice for Geo if he doesnt have to look at both client log and advanced ice log to get ALL logs about ice. The logs coming directly from `IceAdapterImpl` is very low amount compared to STDOUT/ERR anyway.